tests/test_metric_regrouping.js

```diff
@@ -72,90 +75,90 @@ function arrayReduce(dict, element) {
   return dict;
 }
 
-exports['setUp'] = function(test, assert) {
-  parsedObj = JSON.parse(fs.readFileSync('tests/metrics_bundle.json'));
-  assert.ok(parsedObj);
-  test.finish();
-}
+test('setUp', function(t) {
+  parsedObj = require('./metrics_bundle.json');
+  t.ok(parsedObj, 'The fixture was properly loaded.');
+  t.end();
+});
 
-exports['test_counters'] = function(test, assert) {
+test('counters', function(t) {
   var counters = metric_utils.extractCounters(parsedObj),
       names = ['internal.bad_lines_seen', 'internal.packets_received', '3333333.C1s', '3333333.C200ms', '4444444.C10s', '3333333.C29s'],
       visitCount = 0,
       counterDict = counters.reduce(arrayReduce, {});
-  
-  assert.strictEqual(6, counters.length);
-  assert.strictEqual(0, visitCount);
-  
+
+  t.equal(6, counters.length, 'There are 6 counters.');
+  t.equal(0, visitCount, 'The visit count is 0.');
+
   // ensure all the values were copied across.
   names.forEach(function(key) {
-    checkSameCounter(assert, counterDict[key], key);
+    checkSameCounter(t, counterDict[key], key);
     visitCount += 1;
   });
-  
-  assert.strictEqual(6, visitCount);
-  
-  test.finish();
-}
 
-exports['test_gauges'] = function(test, assert) {
+  t.equal(6, visitCount, 'Each counter was represented.');
+
+  t.end();
+});
+
+test('gauges', function(t) {
   var gauges = metric_utils.extractGauges(parsedObj),
       names = ['3333333.G1s', '4444444.G200ms', '3333333.G10s', 'internal.timestamp_lag'],
       visitCount = 0,
       gaugeDict = gauges.reduce(arrayReduce, {});
-  
-  assert.strictEqual(4, gauges.length);
-  assert.strictEqual(0, visitCount);
-  
+
+  t.equal(4, gauges.length, 'There are 4 counters.');
+  t.equal(0, visitCount, 'The visit count is 0.');
+
   names.forEach(function(key) {
-    checkSameGauge(assert, gaugeDict[key], key);
+    checkSameGauge(t, gaugeDict[key], key);
     visitCount += 1;
   });
-  
-  assert.strictEqual(4, visitCount);
-  
-  test.finish();
-}
 
-exports['test_sets'] = function(test, assert) {
+  t.equal(4, visitCount, 'Each gauage was checked.');
+
+  t.end();
+});
+
+test('sets', function(t) {
   var sets = metric_utils.extractSets(parsedObj),
       names = ['4444444.S1s', '3333333.S500ms'],
       visitCount = 0,
       setDict = sets.reduce(arrayReduce, {});
-  
-  assert.strictEqual(2, sets.length);
-  assert.strictEqual(0, visitCount);
-  
+
+  t.strictEqual(2, sets.length, 'There are 2 sets.');
+  t.strictEqual(0, visitCount, 'The visit count is 0.');
+
   names.forEach(function(key) {
-    checkSameSet(assert, setDict[key], key);
+    checkSameSet(t, setDict[key], key);
     visitCount += 1;
   });
-  
-  assert.strictEqual(2, visitCount);
-  
-  test.finish();
-}
 
-exports['test_timers'] = function(test, assert) {
+  t.strictEqual(2, visitCount, 'Each set was checked.');
```

Comment:
I had meant to make them all `t.equal()`. `t.strictEqual()` is an alias for it though.